README.md

```diff
@@ -46,6 +46,20 @@ To get all the latest changes, `janus-core` can also be installed from GitHub:
 python3 -m pip install git+https://github.com/stfc/janus-core.git
 ```
 
+By default, MACE is the only MLIP installed.
```

Comment:
I will add a line on how the user can get the full list of available mlips.

Reply:
I think I might have done this slightly differently to what you meant earlier, but hopefully this is ok?